sass_builder_test.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:build/build.dart';
+import 'package:build_barback/build_barback.dart';
+import 'package:build_test/build_test.dart';
+import 'package:sass_builder/sass_builder.dart';
+import 'package:test/test.dart';
+
+void main() {
+  // These tests only verify which assets are read and written by the
+  // SassBuilder. This is because the inputs are being parsed to find what files
+  // they import.
+  //
+  // These tests do not verify any output as that is determined by the sass
+  // implementation.
+  group('File import parsing tests', () {
+    SassBuilder builder;
+    InMemoryAssetWriter writer;
+    InMemoryAssetReader reader;
+
+    setUp(() {
+      builder = new SassBuilder();
+      reader = new InMemoryAssetReader();
+      writer = new InMemoryAssetWriter();
+    });
+
+    test('no imports, one read and one write', () async {
+      var primary = makeAssetId('a|lib/styles.scss');
+      var inputs = {
+        primary: '/* no imports */',
+      };
+
+      reader.cacheStringAsset(primary, inputs[primary]);
+
+      await runBuilder(builder, inputs.keys, reader, writer, const BarbackResolvers());
```

Comment:
nit: you aren't resolving any code so i think you could pass `null` for the last arg, and not have a dependency on build_barback in this file.